proyecto/src/main/java/com/proyecto/proyecto/ControladorUsuarios.java

```diff
@@ -32,5 +32,19 @@ public String eliminar(@PathVariable("id") int idUsuario, RedirectAttributes att
         return "redirect:/usuarios/index";
     }
 
+    @GetMapping("/unlock/{id}")
+	public String activar(@PathVariable("id") int idUsuario, RedirectAttributes attributes) {		
+    	servicioUsuarios.activar(idUsuario);
+		attributes.addFlashAttribute("msg", "El usuario fue activado y ahora tiene acceso al sistema.");		
+		return "redirect:/usuarios/index";
+	}
+
+    @GetMapping("/lock/{id}")
+	public String bloquear(@PathVariable("id") int idUsuario, RedirectAttributes attributes) {		
+		servicioUsuarios.bloquear(idUsuario);
```

Comment:
#should Creo que todas las operaciones de los usuarios deberían estar protegidas por algún control de manera que no cualquiera pueda realizar estos cambios. Está hecho esto de alguna manera o es algo pendiente?

Reply:
Las operaciones de los usuarios ya están protegidas, solo el administrador tiene acceso. En los archivos DatabaseWebSecurity.java y menu.html se puede apreciar esto.

Reply:
Bárbaro, gracias!